bytecomp/bytelink.ml

```diff
@@ -310,7 +310,7 @@ let link_bytecode ?final_name tolink exec_name standalone =
     ~always:(fun () -> close_out outchan)
     ~exceptionally:(fun () -> remove_file exec_name)
     (fun () ->
-       if standalone then begin
+       if standalone && not !Clflags.noruntime then begin
```

Comment:
I think it would also be correct to skip the DLL checks if the runtime is not being linked - or does `ocamlrun` still use those when invoked directly (@damiendoligez?). In other words, I'm wondering if the call to `link_bytecode` later where `standalone` is `true` should in fact pass `(not !Clflags.noruntime)` instead?

Reply:
> does ocamlrun still use those when invoked directly (@damiendoligez?)

Yes of course. The behaviour of `ocamlrun` does not depend on how it is invoked, with the exception of a bit of command-line mangling.

Reply:
Of course, sorry - I think I was confusing it with the writing of the runtime string for `-use-runtime`, not the sections in the bytecode itself.